### ISSUE TITLE
inconsistency between the prior scale defaults across BF tests

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/parameters#296]
I noticed an inconsistency between the prior scale defaults across tests: 
it is `0.707` for _t_-tests, but unexpectedly changes to `0.333` for correlation test.

I think, just like `BayesFactor` itself, we should be consistent across tests in this regard otherwise the user is going to be caught by surprise. 

``` r
# setup
set.seed(123)
library(parameters)
library(BayesFactor)
#> Loading required package: coda
#> Loading required package: Matrix
#> ************
#> Welcome to BayesFactor 0.9.12-4.2. If you have questions, please contact Richard Morey (richarddmorey@gmail.com).
#> 
#> Type BFManual() to open the manual.
#> ************

# one sample t test
mod1 <- ttestBF(x = rnorm(100, 1, 1))

model_parameters(mod1)
#> Parameter  | Median |       89% CI |   pd | % in ROPE |              Prior | Effects |   Component |    BF
#> ----------------------------------------------------------------------------------------------------------
#> Difference |   0.85 | [0.66, 1.05] | 100% |        0% | Cauchy (0 +- 0.71) |   fixed | conditional | > 999

# independent t test
mod2 <- ttestBF(formula = wt ~ am, data = mtcars)

model_parameters(mod2)
#> Parameter  | Median |       89% CI |   pd | % in ROPE |              Prior | Effects |   Component |    BF
#> ----------------------------------------------------------------------------------------------------------
#> Difference |   1.26 | [0.80, 1.69] | 100% |        0% | Cauchy (0 +- 0.71) |   fixed | conditional | > 999

# paired t test
mod3 <- ttestBF(
  x = sleep$extra[sleep$group == 1],
  y = sleep$extra[sleep$group == 2],
  paired = TRUE
)

model_parameters(mod3)
#> Parameter  | Median |       89% CI |   pd | % in ROPE |              Prior | Effects |   Component |    BF
#> ----------------------------------------------------------------------------------------------------------
#> Difference |   1.72 | [0.62, 3.31] | 100% |        0% | Cauchy (0 +- 0.71) |   fixed | conditional | 17.26

# correlation
mod4 <- correlationBF(y = iris$Sepal.Length, x = iris$Sepal.Width)

model_parameters(mod4)
#> Parameter | Median |        89% CI |     pd | % in ROPE |              Prior | Effects |   Component |   BF
#> -----------------------------------------------------------------------------------------------------------
#> rho       |  -0.11 | [-0.24, 0.00] | 92.12% |    43.27% | Cauchy (0 +- 0.33) |   fixed | conditional | 0.51
```

<sup>Created on 2020-09-09 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

<details>

<summary>Session info</summary>

``` r
devtools::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value                       
#>  version  R version 3.6.3 (2020-02-29)
#>  os       macOS Mojave 10.14.6        
#>  system   x86_64, darwin15.6.0        
#>  ui       X11                         
#>  language (EN)                        
#>  collate  en_US.UTF-8                 
#>  ctype    en_US.UTF-8                 
#>  tz       Europe/Berlin               
#>  date     2020-09-09                  
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package      * version    date       lib source                               
#>  assertthat     0.2.1      2019-03-21 [1] CRAN (R 3.6.0)                       
#>  backports      1.1.9      2020-08-24 [1] CRAN (R 3.6.2)                       
#>  BayesFactor  * 0.9.12-4.2 2018-05-19 [1] CRAN (R 3.6.0)                       
#>  bayestestR     0.7.2.1    2020-08-15 [1] Github (easystats/bayestestR@1ffb88f)
#>  callr          3.4.4      2020-09-07 [1] CRAN (R 3.6.3)                       
#>  cli            2.0.2      2020-02-28 [1] CRAN (R 3.6.0)                       
#>  coda         * 0.19-3     2019-07-05 [1] CRAN (R 3.6.0)                       
#>  crayon         1.3.4      2017-09-16 [1] CRAN (R 3.6.0)                       
#>  desc           1.2.0      2018-05-01 [1] CRAN (R 3.6.0)                       
#>  devtools       2.3.1      2020-07-21 [1] CRAN (R 3.6.3)                       
#>  digest         0.6.25     2020-02-23 [1] CRAN (R 3.6.0)                       
#>  ellipsis       0.3.1      2020-05-15 [1] CRAN (R 3.6.2)                       
#>  evaluate       0.14       2019-05-28 [1] CRAN (R 3.6.0)                       
#>  fansi          0.4.1      2020-01-08 [1] CRAN (R 3.6.2)                       
#>  fs             1.5.0      2020-07-31 [1] CRAN (R 3.6.2)                       
#>  glue           1.4.2      2020-08-27 [1] CRAN (R 3.6.2)                       
#>  gtools         3.8.2      2020-03-31 [1] CRAN (R 3.6.2)                       
#>  highr          0.8        2019-03-20 [1] CRAN (R 3.6.0)                       
#>  htmltools      0.5.0      2020-06-16 [1] CRAN (R 3.6.2)                       
#>  insight        0.9.5      2020-09-07 [1] CRAN (R 3.6.3)                       
#>  knitr          1.29       2020-06-23 [1] CRAN (R 3.6.3)                       
#>  lattice        0.20-41    2020-04-02 [1] CRAN (R 3.6.2)                       
#>  magrittr       1.5        2014-11-22 [1] CRAN (R 3.6.0)                       
#>  Matrix       * 1.2-18     2019-11-27 [2] CRAN (R 3.6.3)                       
#>  MatrixModels   0.4-1      2015-08-22 [1] CRAN (R 3.6.0)                       
#>  memoise        1.1.0      2017-04-21 [1] CRAN (R 3.6.0)                       
#>  mvtnorm        1.1-1      2020-06-09 [1] CRAN (R 3.6.2)                       
#>  parameters   * 0.8.2.1    2020-08-18 [1] Github (easystats/parameters@c8f2f2b)
#>  pbapply        1.4-3      2020-08-18 [1] CRAN (R 3.6.3)                       
#>  pkgbuild       1.1.0      2020-07-13 [1] CRAN (R 3.6.3)                       
#>  pkgload        1.1.0      2020-05-29 [1] CRAN (R 3.6.2)                       
#>  prettyunits    1.1.1      2020-01-24 [1] CRAN (R 3.6.2)                       
#>  processx       3.4.4      2020-09-03 [1] CRAN (R 3.6.3)                       
#>  ps             1.3.4      2020-08-11 [1] CRAN (R 3.6.3)                       
#>  R6             2.4.1      2019-11-12 [1] CRAN (R 3.6.0)                       
#>  Rcpp           1.0.5      2020-07-06 [1] CRAN (R 3.6.3)                       
#>  remotes        2.2.0      2020-07-21 [1] CRAN (R 3.6.3)                       
#>  rlang          0.4.7      2020-07-09 [1] CRAN (R 3.6.3)                       
#>  rmarkdown      2.3        2020-06-18 [1] CRAN (R 3.6.3)                       
#>  rprojroot      1.3-2      2018-01-03 [1] CRAN (R 3.6.0)                       
#>  sessioninfo    1.1.1      2018-11-05 [1] CRAN (R 3.6.0)                       
#>  stringi        1.5.3      2020-09-09 [1] CRAN (R 3.6.3)                       
#>  stringr        1.4.0      2019-02-10 [1] CRAN (R 3.6.0)                       
#>  testthat       2.3.2      2020-03-02 [1] CRAN (R 3.6.2)                       
#>  usethis        1.6.1      2020-04-29 [1] CRAN (R 3.6.2)                       
#>  withr          2.2.0      2020-04-20 [1] CRAN (R 3.6.2)                       
#>  xfun           0.17       2020-09-09 [1] CRAN (R 3.6.3)                       
#>  yaml           2.2.1      2020-02-01 [1] CRAN (R 3.6.0)                       
#> 
#> [1] /Users/patil/Library/R/3.6/library
#> [2] /Library/Frameworks/R.framework/Versions/3.6/Resources/library
```

</details>

**Comment by mattansb** [comment]
Not sure what you expect on the `parameters` side? This is the default behavior of `correlationBF` / `ttestBF` (different tests use differently scales priors...).

**Comment by IndrajeetPatil** [comment]
Ach, apologies. 
I thought we had opinionated priors for these tests, but now that you mention this, I realize this inconsistency stems from `BayesFactor` itself.

**Comment by IndrajeetPatil** [comment]
@mattansb Thinking of this further, another thing I notice is that `Prior_Scale` values are `NA` for `anovaBF` outputs. 

Should we be including `rscaleFixed` and `rscaleRandom` argument values here?

```r
library(BayesFactor)
#> Loading required package: coda
#> Loading required package: Matrix
#> ************
#> Welcome to BayesFactor 0.9.12-4.2. If you have questions, please contact Richard Morey (richarddmorey@gmail.com).
#> 
#> Type BFManual() to open the manual.
#> ************
library(parameters)

data(puzzles)

result = anovaBF(RT ~ shape*color + ID, data = puzzles, whichRandom = "ID",
                 whichModels = 'top', progress=FALSE)

as.data.frame(model_parameters(result))$Prior_Scale
#>  [1] NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA
```

**Comment by mattansb** [comment]
It's hard to match the different parameters to the `rscale*` values (I've had a little back and forth about this with Richard some time ago). That is why, currently, `bayestestR::describe_prior()` behaves like this:

```R
describe_prior(result)
#>    Parameter Prior_Distribution Prior_Location Prior_Scale
#> 1      fixed             cauchy              0   0.5000000
#> 2     random             cauchy              0   1.0000000
#> 3 continuous             cauchy              0   0.3535534
```
(And does not return the dist/loc/scale on a parameter basis...)

It is unfortunate that the docs and internals from `BayesFactor` are somewhat opaque... 🤷‍♂️

I suggest opening this as a feature request on `bayestestR` - if it works there, it will work here as well.
